src/Cpp/1-getting-started/1-3-3-SettingUp3DRendering/1-3-6-Camera.vcxproj

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>16.0</VCProjectVersion>
+    <Keyword>Win32Proj</Keyword>
+    <ProjectGuid>{E38C17E3-29B9-40BF-9372-0EB24DDE4B06}</ProjectGuid>
+    <RootNamespace>My13HelloTriangle</RootNamespace>
```

Comment:
Oi, where did that come from? :o

Reply:
I do not know, multiple projects seem to have that